x-pack/plugins/synthetics/public/apps/synthetics/components/settings/data_retention.tsx

```diff
@@ -76,6 +91,65 @@ export const DataRetentionTab = () => {
   );
 };
 
+const Unprivileged = () => {
+  const {
+    services: { docLinks },
+  } = useKibana();
+  return (
+    <EuiEmptyPrompt
+      data-test-subj="syntheticsUnprivileged"
+      color="plain"
+      icon={<EuiIcon type="logoObservability" size="xl" />}
+      title={
+        <h2>
+          <FormattedMessage
+            id="xpack.synthetics.noFindingsStates.unprivileged.unprivilegedTitle"
+            defaultMessage="Privileges required"
+          />
+        </h2>
+      }
+      body={
+        <p>
+          <FormattedMessage
+            id="xpack.synthetics.params.unprivileged.unprivilegedDescription"
+            defaultMessage="To view Synthetics monitors data usage and retention, you must update privileges. For more information, contact your Kibana administrator. {docsLink}"
```

Comment:
```suggestion
            defaultMessage="You need additional privileges to view Synthetics data usage and retention settings. {docsLink}"
```
Suggesting to simplify the message if the suggestion below is not possible.

If that is possible I would suggest listing the required privileges as bullet points after this message and with a similar formatting instead of at the bottom, then to have the Read more link centered as an empty button, after the list of privileges so it looks more natural:

---
                                      **Missing privileges**
You need additional privileges to view Synthetics data usage and retention settings:

- `read`, `monitor` on the following Elasticsearch indices: `synthetics-*`
- `read_ilm`, `monitor` on the Elasticsearch cluster

                                            **Learn more**